### PR TITLE
Document new metadata group behaviour

### PR DESCRIPTION
**Pull request by ivester**
Relations:
  - Related PRs: https://github.com/livingdocsIO/livingdocs-editor/pull/5249

### CODE REVIEW

File: content/reference-docs/project-config/content-types.md [hunk 1]
```diff
@@ -203,19 +203,23 @@ Get a list of available [metadata plugins]({{< ref "/reference-docs/document/met
 ## Metadata Groups
 *has UI support*
 
-The Metadata Groups allow you to logically group the fields for display on the publish panel.
+The Metadata Groups allow you to logically group the fields into collapsible cards. By default all the cards are expanded but can be configured to be collapsed with the property `metadataGroups.expanded: false`.
```
Comment (ajwild) [inline content/reference-docs/project-config/content-types.md:206]: I'm never sure how to nicely show a property nested within a named array. I'd probably just go with `expanded: false`, but maybe `metadataGroups[].expanded: false` also works. `metadataGroups.expanded` doesn't seems quite right as I'd expect `metadataGroups` to be an object.

File: content/reference-docs/project-config/content-types.md [hunk 2]
```diff
@@ -203,19 +203,23 @@ Get a list of available [metadata plugins]({{< ref "/reference-docs/document/met
...
+Once a card has been manually collapsed/expanded in the UI, then this state will be saved in the local storage of the browser and the card will show in that state even after a page reload.
 
 The schema looks as follows:
 ```js
   metadataGroups: ms.arrayOf(ms.strictObj({
     label: 'string',
-    properties: ms.arrayOf(ms.string())
+    expanded: ms.boolean(), // optional, default is true
+    properties: ms.arrayOf(ms.string()),
+
```
Comment (ajwild) [inline content/reference-docs/project-config/content-types.md:215]: This empty line can be removed.

File: content/reference-docs/project-config/content-types.md [hunk 3]
```diff
@@ -203,19 +203,23 @@ Get a list of available [metadata plugins]({{< ref "/reference-docs/document/met
...
+The Metadata Groups allow you to logically group the fields into collapsible cards. By default all the cards are expanded but can be configured to be collapsed with the property `metadataGroups.expanded: false`.
+Once a card has been manually collapsed/expanded in the UI, then this state will be saved in the local storage of the browser and the card will show in that state even after a page reload.
 
 The schema looks as follows:
 ```js
   metadataGroups: ms.arrayOf(ms.strictObj({
     label: 'string',
-    properties: ms.arrayOf(ms.string())
+    expanded: ms.boolean(), // optional, default is true
+    properties: ms.arrayOf(ms.string()),
```
Comment (ajwild) [inline content/reference-docs/project-config/content-types.md:214]: No need for the trailing comma.

File: content/reference-docs/project-config/content-types.md (hunk 1 above)
Comment (ivester) [reply content/reference-docs/project-config/content-types.md:206]: I like `metadataGroups[].expanded`. I saw it being used in a few other places like this.

**Comment by ivester** [comment]
@ajwild thank you for the review!

**Review by ajwild — APPROVED** [review]

**Pull request merged**